tests/crypto/test_crypto.c

```diff
@@ -8,14 +8,50 @@
 
 #include "blake2b_data.h"
 #include "crypto/iota_crypto.h"
+#include "test_vectors.h"
```

Comment:
please use a more specific file name, like 
```suggestion
#include "ed25519_signature_edges.h"
```